app/forms/duration_form.py

```diff
@@ -1,10 +1,19 @@
+from __future__ import annotations
+
+from collections.abc import Callable
```

Comment:
Minor - but any reason why we're aren't using typing Callable here?

Reply:
Updated  `callable` is now called from `typing`.